test/sanity/validate-modules/ignore.txt

```diff
@@ -162,19 +162,30 @@ lib/ansible/modules/cloud/spotinst/spotinst_aws_elastigroup.py E323
 lib/ansible/modules/cloud/univention/udm_dns_zone.py E322
 lib/ansible/modules/cloud/univention/udm_share.py E322
 lib/ansible/modules/cloud/univention/udm_share.py E323
+lib/ansible/modules/cloud/univention/udm_share.py E324
```

Comment:
This doesn't look right. The original PR only removed ignore entries, yet this one is adding them.

Reply:
OK.